src/query/storages/system/src/processes_table.rs

```diff
@@ -45,6 +45,8 @@ pub struct ProcessesTable {
 impl SyncSystemTable for ProcessesTable {
     const NAME: &'static str = "system.processes";
 
+    const IS_LOCAL: bool = false;
```

Comment:
Need to add the `nodeid` to processes table?

For a query which distributed executing in 2-nodes cluster: query0 and query1:
Does the query id are same in query0 and query1?

Reply:
If they are same no need add the `nodeid` to the processes table.

Reply:
> If they are same no need add the `nodeid` to the processes table.

for a distributed query, it will only show once in processes table.

Reply:
let's keep it consistent with the standalone mode.